Ical.Net/VTimeZoneInfo.cs

```diff
@@ -9,6 +9,7 @@
 using Ical.Net.CalendarComponents;
 using Ical.Net.DataTypes;
 using Ical.Net.Evaluation;
+using Ical.Net.Utility;
```

Comment:
maybe resolve 'VTimeZoneInfo' overrides Object.Equals(object o) but does not override Object.GetHashCode() ?